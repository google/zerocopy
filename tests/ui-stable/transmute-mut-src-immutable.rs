../ui-nightly/transmute-mut-src-immutable.rs