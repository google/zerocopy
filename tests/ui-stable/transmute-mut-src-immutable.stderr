error[E0308]: mismatched types
  --> tests/ui-stable/transmute-mut-src-immutable.rs:17:22
   |
17 |     let _: &mut u8 = transmute_mut!(&0u8);
   |                      ^^^^^^^^^^^^^^^^^^^^
   |                      |
   |                      types differ in mutability
   |                      arguments to this function are incorrect
   |
   = note: expected mutable reference `&mut _`
                      found reference `&u8`
note: function defined here
  --> tests/ui-stable/transmute-mut-src-immutable.rs:17:22
   |
17 |     let _: &mut u8 = transmute_mut!(&0u8);
   |                      ^^^^^^^^^^^^^^^^^^^^
   = note: this error originates in the macro `transmute_mut` (in Nightly builds, run with -Z macro-backtrace for more info)
